react-hyf/react-hyf-client/src/components/previoushomeworks.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import Homework1 from './homework1';
+import Homework2 from './homework2';
+
+const PreviousHomeworks = () => {
+  return (
+    <div className="heading">
+      <h1 className="homework-headings">Homework1</h1>
+      <Homework1 />
+      <h1 className="homework-headings">Homework2</h1>
+      <Homework2 />
+      <h1 className="homework-headings">Homework3</h1>
```

Comment:
Homework 3 isn't part of PreviousHomework right? Better to move this title out of this component ;-)

Reply:
done

> Nice work on your homework @zehrakopar! Noted a few things in your code, and it seems deleting a todo using a click doesn't work anymore in Homework 2

fixed..